qa/cephfs/mount/kclient/overrides/distro/stock/centos_9.stream.yaml

```diff
@@ -0,0 +1 @@
+.qa/distros/all/centos_9.stream.yaml
```

Comment:
Just wondering, isn't centos 9 already covered by another alternative in this directory, `qa/cephfs/mount/kclient/overrides/distro/stock/k-stock.yaml`?  Or is the intent to do more runs with centos 9 than with ubuntu 22.04?

Reply:
The intent of this yaml is to only do stock kernel testing with centos9. The main reason for this was that we can actually downstream fixes for that kernel in a predictable manner.

Reply:
I see, I missed that `k-stock.yaml` is not an alternative but rather the snippets get combined there.  Thanks!